dashboard/app/controllers/reports_controller.rb

```diff
@@ -56,6 +56,30 @@ def all_usage
     render 'usage', formats: [:html]
   end
 
+  def funometer
+    authorize! :read, :reports
+
+    # Compute the global funometer ratio.
+    all_ratings = PuzzleRating.all
+    positive_ratings = all_ratings.where(rating: 1)
+    @global_percentage = 100.0 * positive_ratings.count / all_ratings.count
+
+    # Construct the URL showing the daily trend.
+    percentages_by_day = all_ratings.select('100.0 * SUM(rating) / COUNT(rating)').where('created_at > ?', Time.now.prev_month).group('DATE(created_at)').order('DATE(created_at)').pluck('DATE(created_at)', '100.0 * SUM(rating) / COUNT(rating)')
```

Comment:
This appears to be the daily trend _for the last month_. Can that be specified somewhere?